routes/index.js

```diff
@@ -11,8 +11,8 @@ const constructorMethod = (app) => {
     app.use('/', async (req, res) => {
         if(req.session.user){
             //console.log(await getUserByUserName(req.session.user.username))
-            const user = await getUserByUserName(req.session.user.username)
-            const icon = user.icon
+            // const user = await getUserByUserName(req.session.user.username)
+            const icon = req.session.user.icon
```

Comment:
I don't believe that `req.session.user` has the property `icon`.

Reply:
actually if you have a try, you can find that it works well